### ISSUE TITLE
Pass list of initially toggled options for multi-select menus

### ISSUE THREAD

**Issue by kelvindecosta** [IngoMeyer441/simple-term-menu#28]
Hey!

I'm looking for something like this:

```python
options = ["apple", "banana", "orange"]
menu = TerminalMenu(options, multi_select=True, selected_options=set([0]))
menu.show()
```

... which would show:

```
> [*] apple
  [ ] banana
  [ ] orange
```

Is it possible to provide a multi-select menu with a list of options that are already set to True?

Thanks for maintaining this amazing project!
Any feedback is appreciated!

**Comment by IngoMeyer441** [comment]
Hey @kelvindecosta,
definitely a useful feature request and should not be so much work to implement. I will look into this when I have some free time!

**Comment by kelvindecosta** [comment]
Hey @IngoMeyer441 

Thanks for the quick response!

I was playing around and found another interesting approach for simulating multi-select.

```python
fruits = ["apple", "banana", "orange"]
chosen = set()

while True:
    menu = TerminalMenu([f"""[{'*' if f in chosen else ' '}] {f}""" for f in fruits], clear_screen=True, clear_menu_on_exit=False)

    if (choice := menu.show()) is not None:
        fruit = fruits[choice]  

        if fruit in chosen:
            chosen.remove(fruit)
        else
            chosen.add(first)

        # do something with fruit
        pass
    else:
         break
```

This fits my use case but there is one major annoyance.
Whenever `menu.show()` is called, it re-renders the menu like it should.
However, the cursor resets.

Is there a way I can set the initial cursor position?

Thanks for your time.
Any feedback is appreciated!

**Comment by IngoMeyer441** [comment]
You can pass the initial cursor index with the `cursor_index` constructor parameter.

**Comment by UnstoppableDrew** [comment]
I implemented a pre-select feature but I'm having trouble committing it because the pre-commit hook keeps crashing with:
````stderr:
    Traceback (most recent call last):
      File "/home/amarold/.cache/pre-commit-zipapp/-eYCHjv49dDHr21Lw7EXR3hR9h4GSmNzRdQVGEnG1aw/python", line 48, in <module>
        exit(main())
      File "/home/amarold/.cache/pre-commit-zipapp/-eYCHjv49dDHr21Lw7EXR3hR9h4GSmNzRdQVGEnG1aw/python", line 32, in main
        runpy.run_module(args.m, run_name='__main__', alter_sys=True)
      File "/usr/lib/python3.8/runpy.py", line 207, in run_module
        return _run_module_code(code, init_globals, run_name, mod_spec)
      File "/usr/lib/python3.8/runpy.py", line 97, in _run_module_code
        _run_code(code, mod_globals, init_globals,
      File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
        exec(code, run_globals)
      File "/home/amarold/.cache/pre-commit-zipapp/-eYCHjv49dDHr21Lw7EXR3hR9h4GSmNzRdQVGEnG1aw/wheels/nodeenv-1.6.0-py2.py3-none-any.whl/nodeenv.py", line 48, in <module>
    ModuleNotFoundError: No module named 'pkg_resources'
```

**Comment by IngoMeyer441** [comment]
@UnstoppableDrew Great that you want to contribute! :+1: 
Hmm, I was not aware that the `pre-commit` stand alone app has more dependencies than Python. AFAIK, the error message
```
ModuleNotFoundError: No module named 'pkg_resources'
```
indicates a missing setuptools installation. Which system do you use? On Ubuntu, for example, you could fix this with
```
sudo apt install python3-setuptools
```

**Comment by IngoMeyer441** [comment]
The repository also contains a GitHub action, so you can also push to GitHub to let the checks run.

**Comment by IngoMeyer441** [comment]
Thanks to @UnstoppableDrew this issue is fixed by PR #30.
